db/seeds.rb

```diff
@@ -530,6 +530,19 @@ def create_if_none(klass, count = nil)
 
 ##############################################################################
 
+seeder.create_if_none(Sponsorship) do
+  5.times do
+    Sponsorship.create!(
+      organization: Organization.order(Arel.sql("RANDOM()")).first,
+      user: User.order(Arel.sql("RANDOM()")).first,
+      level: "silver",
```

Comment:
there's a tiny chance this breaks if `random()` returns the same organization twice as levels have to be unique per org

Reply:
Good point! The minimum number of organizations is three, so to avoid this issue I'll just make one for each of the first three organizations. :+1: We can make this generate more diverse data later if we feel it will make life easier.